docs/docs/cloud/aws/compliance.md

```diff
@@ -0,0 +1,204 @@
+| require-vpc-changes-alarm                  | Ensure a log metric filter and |
+|                                            | alarm exist for VPC changes    |
+| require-non-mfa-login-alarm                | Ensure a log metric filter and |
+|                                            | alarm exist for AWS Management |
+|                                            | Console sign-in without MFA    |
+| require-root-user-usage-alarm              | Ensure a log metric filter and |
+|                                            | alarm exist for usage of root  |
+|                                            | user                           |
+| require-iam-policy-change-alarm            | Ensure a log metric filter     |
+|                                            | and alarm exist for IAM policy |
+|                                            | changes                        |
+| require-cloud-trail-change-alarm           | Ensure a log metric filter     |
+|                                            | and alarm exist for CloudTrail |
+|                                            | configuration changes          |
+| require-console-login-failures-alarm       | Ensure a log metric filter and |
+|                                            | alarm exist for AWS Management |
+|                                            | Console authentication         |
+|                                            | failures                       |
+| require-cmk-disabled-alarm                 | Ensure a log metric filter and |
+|                                            | alarm exist for disabling or   |
+|                                            | scheduled deletion of customer |
+|                                            | managed keys                   |
+| require-s3-bucket-policy-change-alarm      | Ensure a log metric filter     |
+|                                            | and alarm exist for S3 bucket  |
+|                                            | policy changes                 |
+| require-config-configuration-changes-alarm | Ensure a log metric filter     |
+|                                            | and alarm exist for AWS Config |
+|                                            | configuration changes          |
+| no-public-ingress-sgr                      | An ingress security group rule |
+|                                            | allows traffic from /0.        |
++--------------------------------------------+--------------------------------+
+
+## CLI Commands
+
+Scan for misconfigurations in an AWS account based on AWS CIS 1.2 benchmark:
+
+```
+$ trivy aws  --compliance=awscis1.2 
+```
+
+![img.png](../../../imgs/aws-cis-report.png)
+
+
+You can also summarize the report to get a full compliance report with all the included checks.
+```shell
+$ trivy aws --compliance=awscis1.2 --report=summary
+```
+![img.png](../../../imgs/cloud-compliance-summary.png)
```

Comment:
I prefer text-based as it is easy to update. Do you have any reason you want to use an image here?